src/components/SourceFooter.css

```diff
@@ -12,50 +12,39 @@
   user-select: none;
 }
 
-html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+.source-footer__command-bar {
```

Comment:
Hey @Garbee, do you think you could style this without BEM for now and make a new issue for adopting BEM with this code snippet there.

I think the consensus is that we're in favor of BEM, but would like to come up with a plan so the project adopts BEM well.